README.md

```diff
@@ -61,6 +179,23 @@ Binding preference is determined by the relative orientation of the first and la
 | &rarr; | &darr; | G |
 | &rarr; | &larr; | T |
 
+
+### Authors' Note: 
+A number of ambiguities present in the original program are given defined behavior in this rendition of the system, listed below. 
+
+**Binding Preference of Single Amino Acid Enzymes**
+
+Since the binding preference of an enzyme is determined by the orientation of its first and last element, enzymes which consist of only a single instruction don't have a clearly defined 
```

Comment:
```suggestion
Since the binding preference of an enzyme is determined by the relative orientation of its first and last amino acids, enzymes which consist of only a single amino acid don't have a clearly defined 
```

Reply:
Unsure about these edits. See above comment.

Reply:
I think it's a good change, more correct language use.